src/pages/home/report/ReportActionCompose/SilentCommentUpdater.js

```diff
@@ -52,7 +56,9 @@ function SilentCommentUpdater({comment, commentRef, report, value, updateComment
     useEffect(() => {
         // Value state does not have the same value as comment props when the comment gets changed from another tab.
         // In this case, we should synchronize the value between tabs.
-        const shouldSyncComment = prevCommentProp !== comment && value !== comment;
+
+        // Adding isWeb check to fix issue on Android, assuming we don't need tab sync on mobiles - https://github.com/Expensify/App/issues/28562
+        const shouldSyncComment = isWeb && prevCommentProp !== comment && value !== comment;
```

Comment:
Ah sorry for not mentioning this earlier, but ideally we don't use `getPlatform` and instead create a specificfile for the web-specific code